card/mod/account/set/right/account.rb

```diff
@@ -57,12 +57,27 @@ def token_path_opts
 
 format :html do
   view :raw do
+    # FIXME: use field_nest instead of parsing content
+    #        Problem: whey you do that then the fields are missing in the sign up form
+    # output(
+    #   [
+    #     field_nest(:email, view: :titled, title: "email"),
+    #     field_nest(:password, view: :titled, title: "password")
+    #   ]
+    # )
     %({{+#{Card[:email].name}|titled;title:email}}
-     {{+#{Card[:password].name}|titled;title:password}})
+      {{+#{Card[:password].name}|titled;title:password}})
   end
 
   view :edit do
     voo.structure = true
+    voo.edit_structure = [[:email, "email"], [:password, "password"]]
```

Comment:
The previous implementation didn't work in edit mode with +*account nested in a structure.